### ISSUE TITLE
Fixable error in sjt.df

### ISSUE THREAD

**Issue by freecube** [sjPlot/devel#91]
I received the following error using sjt.df:

Error in as.data.frame.default(x[[i]], optional = TRUE, stringsAsFactors = stringsAsFactors) : 
  cannot coerce class ""describe"" to a data.frame

After a bit of snooping, I realized that the describe function of the psych library had been updated since sjt.df was created and it was no longer compatible. To fix this issue, simply replace the following code

        mydf <- round(describe(mydf), 2)
        mydf <- data.frame(mydf[, 1:2], missings, missings.percentage, 
            mydf[, 3:ncol(mydf)])

with this:

        mydf <- round(describe(mydf), 2)
        mydf <- do.call(cbind, mydf)
        mydf <- data.frame(mydf[, 1:2], missings, missings.percentage, 
            mydf[, 3:ncol(mydf)])

Best,

**Comment by sjPlot** [comment]
I guess that not specifying the package when calling `describe` may use a same named function from another packag? I don't get any errors, however, I have to replace `describe` in the source with `psych::describe`.

Could you please check whether `??describe` finds any other packages with this function? (_Hmisc_ or _testthat_ for example do have functions names `describe` as well).

I suppose this bug is fixed when addind `psych::` to the source, I'll commit an update soon.

**Comment by freecube** [comment]
I had tried to use psych::describe in my code and I still got the same
issue. I googled the error and got an issue with the psych::describe
function, which had been changed since it worked for the user last. Is your
psych package the current version? If not, try updating it and see if you
get the error.

On Tuesday, January 19, 2016, Daniel Lüdecke <notifications@github.com>
wrote:

> Closed #91 <https://github.com/sjPlot/devel/issues/91>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sjPlot/devel/issues/91#event-518986030>.
>


-- 
Grant S. Shields, M.A.
University of California, Davis
Department of Psychology | Ph.D. Student
